src/Domain.jl

```diff
@@ -1603,6 +1603,18 @@ end
     end
 end
 
+@inline function jacobianynsderiv!(jac::S,domain::Union{ConstantTPDomain,ParametrizedTPDomain},rxnarray::Array{Int64,2},efficiencyinds::Array{I,1},cs::Array{Float64,1},kfs::Array{Float64,1},krevs::Array{Float64,1},T::Float64,V::Float64,C::Float64) where {S<:AbstractArray,I<:Integer}
```

Comment:
Can you squash the "Add helper function jacobianynsderiv! that deals with ns partials for" commits?